setup.cfg

```diff
@@ -17,3 +17,13 @@
 # Generated by synthtool. DO NOT EDIT!
 [bdist_wheel]
 universal = 1
+[pytype]
+python_version = 3.8
+inputs =
+    google/cloud/
+exclude =
+    tests/
+    google/cloud/bigquery_v2/
+output = .pytype/
+# Workaround for https://github.com/google/pytype/issues/150
+disable = pyi-error
```

Comment:
This file is synthtool generated, but for the reference and pass current tests changed the current file.

Reply:
https://github.com/google/pytype/issues/150 is marked as closed. Can you provide a little more context as to why this is needed?